### DISCUSSION TITLE
when  cmake,  -- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed

### DISCUSSION THREAD

**Discussion by 123xyl** [openmopac/mopac#102] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/96686347/183273571-6e0b786e-ea24-41b8-8ace-84883bde16ee.png)

**Comment by susilehtola** [comment, marked as answer]
This is a successful configure.

**Reply by 123xyl** [reply]
Thank you！
